app/validators/strategyValidator.js

```diff
@@ -61,4 +61,70 @@ const createStrategyValidatorSchema = {
     }
 };
 
-module.exports = { createStrategyValidatorSchema };
\ No newline at end of file
+const updateStrategyValidatorSchema = {
+    type: {
+        notEmpty: { errorMessage: 'type is optional' },
```

Comment:
This statement seems confusing. If notEmpty is used, it implies that this field is required; however, your message indicates that it is optional.  There are several other fields below also set up like this.

Reply:
> This statement seems confusing. If notEmpty is used, it implies that this field is required; however, your message indicates that it is optional. There are several other fields below also set up like this.

Thanks for pointing out. I had typo here by mistaking character replacement.